app/controllers/api/discussions_controller.rb

```diff
@@ -2,22 +2,16 @@ class API::DiscussionsController < API::RestfulController
   load_and_authorize_resource only: [:show, :mark_as_read, :set_volume]
   load_resource only: [:create, :update]
 
-  def inbox_by_date
-    load_and_authorize_group if params[:group_id]
+  def discussions_for_dashboard
     @discussions = page_collection inbox_threads
     respond_with_discussions
   end
 
-  def inbox_by_organization
+  def discussions_for_inbox
```

Comment:
We had/have a similar situation with 'motions' vs 'proposals', and kinda
settled on calling them 'proposals' in the angular app, and still 'motions'
on the rails side because changing all the names in the existing app is
time-consuming, error-prone, and doesn't really do anything necessarily. (I
tried a full-on rename of Motion => Proposal for a bit a while back, and
nah not worth it.)

On Mon, May 4, 2015 at 12:06 PM, Richard D. Bartlett <
notifications@github.com> wrote:

> In app/controllers/api/discussions_controller.rb
> <https://github.com/loomio/loomio/pull/2137#discussion_r29566298>:
>
> >      @discussions = page_collection inbox_threads
> >      respond_with_discussions
> >    end
> >
> > -  def inbox_by_organization
> > +  def discussions_for_inbox
>
> Yeah fair enough. Just FYI, my preference is to use 'threads' as much as
> possible, because I assume the threads of the future will have lots of
> non-discussion components. But I have no desire to argue the case. Carry on
> :)
>
> eReply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/2137/files#r29566298>.
>